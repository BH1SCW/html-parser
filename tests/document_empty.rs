use html_parser::prelude::*;
use indoc::indoc;
use insta::assert_debug_snapshot;

#[test]
fn it_can_parse_empty_document() -> Result<()> {
    let markup = "";
    let ast = Ast::parse(markup)?;
    assert_debug_snapshot!(ast);
    Ok(())
}
