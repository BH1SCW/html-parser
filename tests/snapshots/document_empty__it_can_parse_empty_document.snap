---
source: tests/document_empty.rs
expression: ast
---
Ast {
    tree_type: Empty,
    nodes: [],
}
